### PR TITLE
LPS-113682 Show in a breadcrumb which page a user is visiting

### PR DESCRIPTION
**Pull request by LuismiBarcos**

### CODE REVIEW

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -206,3 +206,22 @@
...
+	}
+
+	.breadcrumb-item + .breadcrumb-item {
+		padding-left: 0.7rem;
+	}
+	.questions-breadcrumb-unstyled {
+		padding: 0 0.3rem;
+	}
+	.questions-breadcrumb-unstyled:hover {
+		background-color: rgba(39, 40, 51, 0.06);
```
Comment (nhpatt) [inline modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:222]: Mmm... is it not available in clay?

Comment (nhpatt) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:222]: Emiliano typically adds a comment with the name of the color in clay

**Comment by LuismiBarcos** [comment]
Right now, the breadcrumb only supports two levels (current section and its father). I am waiting for the design of the general use of Questions

File: modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss (hunk 1 above)
Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:222]: There is no comment for the background color in this case and is not the `$secondary` color

Comment (LuismiBarcos) [reply modules/apps/questions/questions-web/src/main/resources/META-INF/resources/css/main.scss:222]: Maybe we can use a variable of clay? https://clayui.com/docs/css/color.html#secondary

**Comment by nhpatt** [comment]
ci:forward

**Pull request closed without merging**